x-pack/plugins/observability/public/update_global_navigation.tsx

```diff
@@ -37,15 +42,18 @@ export function updateGlobalNavigation({
           ...link,
           navLinkStatus: someVisible ? AppNavLinkStatus.visible : AppNavLinkStatus.hidden,
         };
-      case 'slos':
+      case 'rules':
         return {
           ...link,
           navLinkStatus: someVisible ? AppNavLinkStatus.visible : AppNavLinkStatus.hidden,
         };
-      case 'rules':
+      case 'slos':
         return {
           ...link,
-          navLinkStatus: someVisible ? AppNavLinkStatus.visible : AppNavLinkStatus.hidden,
+          navLinkStatus:
+            someVisible && capabilities[sloFeatureId].read
```

Comment:
Does the SLO appear ONLY if we have at least one Observability capability?

Reply:
I took the same approach as Cases: a solution needs to be visible and the slo feature needs to be available (read/write). But let me check what I missed in the description